presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -232,7 +225,7 @@ public void testCreateTableAsSelect()
         assertCreateTableAsSelect(
                 Session.builder(getSession()).setSystemProperty("redistribute_writes", "true").build(),
                 "test_union_all",
-                "SELECT orderdate, orderkey, totalprice FROM orders UNION ALL " +
+                "SELECT CAST(orderdate AS DATE) orderdate, orderkey, totalprice FROM orders UNION ALL " +
```

Comment:
why needed?

Reply:
Cassandra does not support DATE and so `orderdate` would be varchar otherwise. For other connectors, the CAST is just no-op.

Reply:
Please add a comment before anybody blindly removes the no-op cast.